.werft/installer-tests.ts

```diff
@@ -364,12 +364,12 @@ export async function installerTests(config: TestConfig) {
     }
 }
 
-function runIntegrationTests() {
+async function runIntegrationTests() {
     werft.phase("run-integration-tests", "Run all existing integration tests");
     const slackAlerts = new Map<string, string>([]);
     for (let test in TESTS) {
         const testPhase = TESTS[test];
-        const ret = callMakeTargets(testPhase.phase, testPhase.description, testPhase.makeTarget);
+        const ret = await callMakeTargets(testPhase.phase, testPhase.description, testPhase.makeTarget);
```

Comment:
How about we remove the `await` here so multiple tests can run in parallel?

Reply:
I'd like to! We can't do that _yet_; test configurations run jobs in serial and depend on that for ordering, such as the ordering between the cert-manager helm chart and the cluster issuer. In the future we can change test definitions to take `Array<Phase>` to define steps that can be invoked in parallel.

Since Terraform already creates resources in parallel we probably won't see a significant speed boost from parallelism but this is an area we should generally explore.